src/CamelKNodeProvider.ts

```diff
@@ -196,7 +196,8 @@ export class CamelKNodeProvider implements vscode.TreeDataProvider<TreeNode> {
 			let o = JSON.parse(temp);
 			for (var i=0; i<o.items.length;i++) {
 				var integrationName = o.items[i].metadata.name;
-				let newNode = new TreeNode("string", integrationName, 'Running', vscode.TreeItemCollapsibleState.None);
+				var status = o.items[i].status.phase;
```

Comment:
I know it was already a problem before but when i read this code again, i really feel that it is so weak. We have no type-safety and neither automated test. If what is returned is modified we will not detect it.

Reply:
created https://github.com/camel-tooling/vscode-camelk/issues/63 and https://github.com/camel-tooling/vscode-camelk/issues/62

Reply:
We have no way to do integration testing at this time. Maybe we can mock it?

Reply:
mocking will allow to test our code logic but not to detect that the returned values have changed.